test/python/WMCore_t/WMRuntime_t/Scripts_t/SetupCMSSWPset_t.py

```diff
@@ -91,7 +91,7 @@ def loadProcessFromPSet(self, psetPath=None):
 
         if psetPath is None:
             psetPath = self.testDir
-        with open(os.path.join(psetPath, "PSet.pkl")) as f:
+        with open(os.path.join(psetPath, "PSet.pkl"), "rb") as f:
```

Comment:
FYI @khurtado I had to change how a pickle file is opened in order to be able to read it from python3. Just pointing this out in case you stumble upon such a problem with pickle in the future! :)